fabric-gamerule-api-v1/src/main/java/net/fabricmc/fabric/impl/gamerule/RuleKeyInternals.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.gamerule;
+
+import net.fabricmc.fabric.api.gamerule.v1.CustomGameRuleCategory;
+
+public interface RuleKeyInternals {
+	/* @Nullable */
+	CustomGameRuleCategory fabric_getCustomCategory();
```

Comment:
The namespacing on these methods are not needed as the method descriptors contain a unique type

Reply:
As far as I can tell, the prefixing on internal methods is standard in Fabric API.